### DISCUSSION TITLE
Where to find simple development case except 'examples' in official website?

### DISCUSSION THREAD

**Discussion by lipxisi** [idaholab/moose#23798] [Q&A Getting Started] [answered]
Hello
When learning MOOSE, the 'examples' in the official website are relatively simple, while the cases in the paper are mostly not open source and complicated,  so Where to find simple development case except 'examples' in official website? 
Thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

did you follow the tutorials?
After those, one should be able to get by with the doxygen and taking other developed objects as examples 
What specifically do you need more info on?

Guillaume
